src/components/FormikMultiSelect.jsx

```diff
@@ -43,7 +43,7 @@ function FormikMultiSelect(props) {
         }
         MenuProps={MenuProps}>
         {Object.entries(options).map(([val, stringRep]) => (
-          <MenuItem key={val} value={val}>
+          <MenuItem key={val} value={parseInt(val) || val}>
```

Comment:
the grades were being stored as strings instead of integers. 
Instead of doing this, I can use `GradeLevel.keys()` and then use `GradeLevel.toString(key)` for the value. 
I would also have to add the `keys()` method for the `Ethnicity` type but we would avoid parsing the integer here and in other places.